src/features/oam-auth/readme.md

```diff
@@ -0,0 +1,108 @@
+# OAM Authentication Feature
```

Comment:
can we find more general abstraction here?
like "google auth for iframed apps" or "oauth proxy for iframe"
if not really and it only works for oam prod then maybe hardcoding urls within the feature's code itself is ok?

@coderabbitai what do you think here?